src/components/toolbar/mdc-toolbar.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <header class="mdc-toolbar">
+    <!-- here are some sugar slots -->
+    <div class="mdc-toolbar__row" v-if="$slots.icon || $slots.title" ref="firstRow">
+      <section class="mdc-toolbar__section mdc-toolbar__section--align-start" v-if="$slots.icon">
+        <slot name="icon" />
+      </section>
+      <section class="mdc-toolbar__section mdc-toolbar__section--align-start" v-if="$slots.title">
+         <slot name="title" />
+      </section>
+    </div>
+    <!-- if the user doesn't want to use the sugar slots then they can inject their own content here -->
+    <slot/>
+  </header>
+</template>
+
+<script lang="babel">
+  import MDCToolbarFoundation from '@material/toolbar/foundation'
+  import * as util from '@material/toolbar/util'
+
+  export default {
+    name: 'Toolbar',
+    props: ['titleElement', 'flexibleRowElement', 'fixedAdjustElement', 'firstRowElement'],
```

Comment:
Not good to pass elements as props. It's against vue guide recommandations and the vue guideline. And it's kind of risky. how do you know elements will be valid at the time vue renders?